src/output.rs

```diff
@@ -37,21 +38,23 @@ pub fn handle_hook_error(err: Error, shellpid: u32, silent: bool) -> i32 {
 }
 
 pub fn print_activation(activated: bool, features: HashSet<Feature>) {
-    if activated {
-        if features.len() == 0 {
-            eprint!("\x1b[1;34mactivated {}", SHADOWENV);
+    if is(Stream::Stderr) {
```

Comment:
🚲🏠: I think we should also rename `print_activation` to `print_activation_to_tty`.